docs/UserGuide.md

```diff
@@ -182,154 +180,309 @@ Example:
 
-Edit task details for GrAB3, including task name, date, duration and tag.
+Edit user task details for GrAB3, including task name, duration, day, week and tag.
 
-Format: `edittask i/INDEX n/TASKNAME sd/DATE d/DURATION`
+Format: `editusertask id/TASKID [tn/TASKNAME] [du/DURATION] [da/DAY] [w/WEEK] [t/TAG]`
 
-- The index must be a **positive integer** and must be **valid**
-- `DURATION` must be in hours
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Items in square brackets are optional, but please input at least 1 value to change.
+</div>
+
+- `DURATION` will be in hours and less than 24 hours
+- `DAY` have to be an integer within 1-7, with 1 being monday and 7 being sunday
+- `WEEEK` have to be a valid positive integer. Total number of week calculate from semester start and end date. <br/>
+    e.g. If the semester is from 01 to 25/09/2020 <br/>
+            * Total week in the semester : 4 <br/>
+            * Week 1 will be from 01 to 06/09/2020 <br/>
+    `editusertask id/usrta001 da/2 w/1` will edit user task to Tuesday of Week 1. 
+- Addition of a user task to a specific day will be **rejected** if the total tasks for that day exceed 24 hours
 
 Example:
-* `edittask i/2 n/Complete 2103 Project sd/20-09-2020 d/3`
-* `edittask i/2 sd/20-09-2020`
-* `edittask i/2 n/Complete 2103 Project d/4`
+* `editusertask id/2 tn/Complete 2103 Project sd/20-09-2020 dy/3`
+* `edituseretask id/2 sd/20-09-2020`
+* `editusertask id/2 tn/Complete 2103 Project da/4`
 
-### Listing All Persons : `list`
+### Indicate that User Task is Completed : `doneusertask`
 
-Shows a list of all persons in the address book.
+Indicate that the user task have been completed.
 
-Format: `list`
+Format: `doneusertask id/TASKID`
 
-### Editing a Person : `edit`
+Example:
+* `doneusertask id/usrta001`
 
-Edits an existing person in the address book.
+### Delete User Task from Task Schedule : `deleteusertask`
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Remove and delete user task from task schedule.
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+Format: `deleteusertask id/TASKID`
 
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+Example:
+* `deleteusertask id/usrta001`
+
+### List the Task Schedule : `listtaskschedule`
```

Comment:
Maybe can change to List all tasks instead of Listing a Task Schedule? As schedule itself refers to a list of tasks

Reply:
I think this one just leave it, easier to understand for the user in my view